sunkit_pyvista/tests/test_magex.py

```diff
@@ -0,0 +1,68 @@
+import astropy.constants as const
+import astropy.units as u
+import matplotlib.pyplot as plt
+import numpy as np
+import pytest
+import pyvista as pv
+import sunpy.map as smap
+from astropy.coordinates import SkyCoord
+from matplotlib import colors
+
+from sunkit_pyvista import SunpyPlotter
+
+pfss = pytest.importorskip("sunkit_magex.pfss")
+from sunkit_magex.pfss import tracing  # noqa: E402
+from sunkit_magex.pfss.sample_data import get_gong_map  # noqa: E402
```

Comment:
```suggestion
from sunkit_magex.pfss import tracing
from sunkit_magex.pfss.sample_data import get_gong_map
```

Reply:
Nope, these need to be below the pytest import or skip or it wont work.